samples/sp_whoIsActive/tasks/buildtasks.js

```diff
@@ -91,33 +71,3 @@ gulp.task("watch", function() {
 				config.paths.project.root + '/test/**/*.ts'],
 		gulp.series('build'));
 });
-
-gulp.task('test', (done) => {
-	let workspace = process.env['WORKSPACE'];
-	if (!workspace) {
-		workspace = process.cwd();
-	}
-	process.env.JUNIT_REPORT_PATH = workspace + '/test-reports/ext_xunit.xml';
-
-	let azuredatastudioPath = 'azuredatastudio';
-	if (process.env['VSCODE_DEV']) {
-		let suffix = os.platform === 'win32' ? 'bat' : 'sh';
```

Comment:
Looks like you should be able to remove the os and cproc imports with the removal of this code